logistics_project/apps/malawi/templatetags/malawi_warehouse_tags.py

```diff
@@ -3,16 +3,17 @@
 from django.core.exceptions import ObjectDoesNotExist
 from logistics.models import SupplyPoint
 from logistics.templatetags.logistics_report_tags import r_2_s_helper
+from logistics.util import config
```

Comment:
ok, i can do that going forward if you prefer - the only reason i was doing it this way was because i had seen it in most places in the code